src/test/java/org/folio/processing/events/handlers/UpdateInstanceEventHandler.java

```diff
@@ -0,0 +1,45 @@
+package org.folio.processing.events.handlers;
+
+import io.vertx.core.json.JsonObject;
+import org.folio.ActionProfile;
+import org.folio.DataImportEventPayload;
+import org.folio.processing.events.services.handler.EventHandler;
+
+import java.util.concurrent.CompletableFuture;
+
+import static org.folio.ActionProfile.Action.UPDATE;
+import static org.folio.ActionProfile.FolioRecord.INSTANCE;
+import static org.folio.DataImportEventTypes.DI_INVENTORY_INSTANCE_UPDATED;
+import static org.folio.rest.jaxrs.model.ProfileSnapshotWrapper.ContentType.ACTION_PROFILE;
+
+public class UpdateInstanceEventHandler implements EventHandler {
+
+  public static final String POST_PROC_INIT_EVENT = "DI_INVENTORY_INSTANCE_UPDATED_READY_FOR_POST_PROCESSING";
```

Comment:
Please do not cut off the word PROCESSING to PROC, it looks weird and is a bit confusing